src/Microsoft.AspNetCore.Server.IntegrationTesting/Deployers/IISExpressDeployer.cs

```diff
@@ -57,6 +68,34 @@ private CancellationToken StartIISExpress(Uri uri, string contentRoot)
                 // Pass on the applicationhost.config to iis express. With this don't need to pass in the /path /port switches as they are in the applicationHost.config
                 // We take a copy of the original specified applicationHost.Config to prevent modifying the one in the repo.
 
+                if (DeploymentParameters.ServerConfigTemplateContent.Contains("[ANCMPath]"))
+                {
+                    string ancmPath;
+                    if (!IsWin8orLater)
+                    {
+                        // The nupkg build of ANCM does not support Win7. https://github.com/aspnet/AspNetCoreModule/issues/40.
+                        ancmPath = @"%ProgramFiles%\IIS Express\aspnetcore.dll";
+                    }
+                    // Bin deployed by Microsoft.AspNetCore.AspNetCoreModule.nupkg
+                    else if (DeploymentParameters.RuntimeFlavor == RuntimeFlavor.CoreClr
+                        && DeploymentParameters.ApplicationType == ApplicationType.Portable)
+                    {
+                        ancmPath = Path.Combine(contentRoot, @"runtimes\win7-x64\native\aspnetcore.dll");
```

Comment:
aspnetcore.dll is different, its bitness needs to match IIS/Express, not the bitness of the OS or app. That said, I need to follow up with sourabh about this if we're ever going to test mixing bitnesses, I think the non-portable apps will deploy the wrong binary.